### ISSUE TITLE
require some babel parser configuration

### ISSUE THREAD

**Issue by beenotung** [mistlog/typedraft#17]
I tried to use typedraft on a file that is using decorator function, got error complaining about babel parser configuration. What is the suggested way to deal with this issue?

Typedraft source  code:
```typescript
@Injectable()
export class CoreService {
  ...
}
```

Error message:
```
$ node_modules/.bin/td src/core/core.service-2.tsx
(node:823896) UnhandledPromiseRejectionWarning: SyntaxError: This experimental syntax requires enabling one of the following parser plugin(s): 'decorators-legacy, decorators' (32:0)
    at Object._raise (/path/to/project/node_modules/.pnpm/@babel/parser@7.10.4/node_modules/@babel/parser/lib/index.js:757:17)
    at Object.raiseWithData (/path/to/project/node_modules/.pnpm/@babel/parser@7.10.4/node_modules/@babel/parser/lib/index.js:750:17)
    at Object.expectOnePlugin (/path/to/project/node_modules/.pnpm/@babel/parser@7.10.4/node_modules/@babel/parser/lib/index.js:8849:18)
    at Object.parseDecorator (/path/to/project/node_modules/.pnpm/@babel/parser@7.10.4/node_modules/@babel/parser/lib/index.js:11385:10)
    at Object.parseDecorators (/path/to/project/node_modules/.pnpm/@babel/parser@7.10.4/node_modules/@babel/parser/lib/index.js:11367:30)
    at Object.parseStatement (/path/to/project/node_modules/.pnpm/@babel/parser@7.10.4/node_modules/@babel/parser/lib/index.js:11200:12)
    at Object.parseBlockOrModuleBlockBody (/path/to/project/node_modules/.pnpm/@babel/parser@7.10.4/node_modules/@babel/parser/lib/index.js:11778:25)
    at Object.parseBlockBody (/path/to/project/node_modules/.pnpm/@babel/parser@7.10.4/node_modules/@babel/parser/lib/index.js:11764:10)
    at Object.parseTopLevel (/path/to/project/node_modules/.pnpm/@babel/parser@7.10.4/node_modules/@babel/parser/lib/index.js:11134:10)
    at Object.parse (/path/to/project/node_modules/.pnpm/@babel/parser@7.10.4/node_modules/@babel/parser/lib/index.js:12836:10)
(node:823896) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:823896) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by mistlog** [comment]
Thanks for reporting this.

It can be fixed by adding `["decorators",{ "decoratorsBeforeExport": true }]` in https://github.com/mistlog/typedraft/blob/0.2.2/src/common/utility.ts#L38.

i.e.

```diff
return parse(code, {
      sourceType: "module",
-     plugins: ["typescript", "jsx", "optionalChaining"],
+     plugins: ["typescript", "jsx", ["decorators",{ "decoratorsBeforeExport": true }]],
 });
```

I will fix it in next release.

**Comment by mistlog** [comment]
now it's fixed in 0.2.3
